code/modules/martial_arts/combos/sleeping_carp/gnashing_teeth.dm

```diff
@@ -0,0 +1,18 @@
+/datum/martial_combo/sleeping_carp/gnashing_teeth
+	name = "Gnashing Teeth"
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "Every second consecutive punch deals extra damage, and you will shout to strike fear into your opponent's heart."
```

Comment:
Is this a leftover from something? It seems to just do a fixed amount of damage now.